src/main/java/modfest/teamgreen/content/block/CrimsonBlocks.java

```diff
@@ -41,7 +45,11 @@ private CrimsonBlocks(String id, Function<Block.Settings, Block> constructor, Fa
 	private CrimsonBlocks(String id, Function<Block.Settings, Block> constructor, FabricBlockSettings settings, Item.Settings itemSettings) {
 		Identifier identifier = CrimsonInit.from(id);
 		this.block = Registry.register(Registry.BLOCK, identifier, constructor.apply(settings.build()));
-		this.item = Registry.register(Registry.ITEM, identifier, new BlockItem(this.block, itemSettings));
+		if (itemSettings == null) {
```

Comment:
new line before if statement

Reply:
also why null item thocc

Reply:
Flower pots don't have an item.

Reply:
Added the missing new line in 9f64c8574f19b61a662f9ecfd49a927533642067.